src/utils/trust/mapper.to.page.ts

```diff
@@ -0,0 +1,47 @@
+import * as Trust from 'model/trust.model';
+import * as Page from 'model/trust.page.model';
+
+const mapDetail = (
```

Comment:
naming - can you rename mapDetail function name to 'mapToTrustDetails' as the function will return a TrustDetails object.

Can you also do the same for mapBeneficialOwner - 'mapToTrustBeneficialOwner'